powsybl-geo-data-model/src/main/java/infrastructure/LineGraphic.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package infrastructure;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.powsybl.iidm.network.Country;
+import com.powsybl.iidm.network.Line;
+import lombok.*;
+import org.apache.commons.lang3.time.StopWatch;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import utils.GeoDataUtils;
+
+import java.awt.Color;
+import java.util.*;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
+ */
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Builder
+public class LineGraphic {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LineGraphic.class);
+
+    private  String id;
+
+    private  int drawOrder;
```

Comment:
Is it really useful?

Reply:
i think we can delete it so let the end user decide what is the correct draw order (or line thickness) for each line.